### DISCUSSION TITLE
Thank you for this card

### DISCUSSION THREAD

**Discussion by Gtasian** [PRProd/HA-Firemote#190] [General]
It's rare to see a Home Assistant card so well designed and integrated, I just wanted to thank the author for this amazing project. I wish I stumbled across this card sooner; it would have saved me a lot of time. The documentation alone was more helpful than any of the official Android integrations.

**Comment by PRProd** [comment]
Thank you for your kind words @Gtasian!  You made my day!

**Comment by Gtasian** [comment]
It has been a struggle getting my Nvidia Shield integrated with full controls. The official integrations and documents didn't make things any easier. I spent hours trying to make a universal media player like the docs recommended only to find out that it doesn't work in the way one would expect it to. I kept searching for "android tv" related things so I never saw your page until I searched for "shield remote". It was exactly what I was looking for and more! Setting this up had to be one of the easiest processes I have seen in Home Assistant. I tried making something like this for my Harmony remote years ago but the furthest I got was a picture elements card and abandoned it shortly after realizing it wouldn't work. I look forward to your future Home Assistant projects.

**Reply by PRProd** [reply]
Thank you for sharing that with me @Gtasian!  After first sharing this project and then deciding to take it further, my goals have been:
 - Ease of use
 - Clear, yet robust documentation
 - Listening to feedback from the community
 
Your comments have been very reassuring that I'm accomplishing my goals, and for that, I THANK YOU!

I've tried to share my project in as many forums as possible, as well as making it (hopefully) easy to find on HACS.  If you happen across someone else who might be looking for something like this, don't be afraid to send them a link to this project!  

I'm in it mainly to make people happy, and to collect some github stars along the way.  I actually set up a HA automation that plays a happy little tune every time this project gets a new star.  :smile:
